src/main/java/it/pagopa/transactions/utils/NodoOperations.java

```diff
@@ -154,6 +159,12 @@ private Mono<String> nodoActivationForUnknownPaymentRequest(
                                     && "PPT_MULTI_BENEFICIARIO"
                                             .equals(nodoAttivaRPTRResponse.getFault().getFaultCode());
 
+                            log.info(
+                                    "Esito nodo attiva RPT. RPT id: [{}] outcome: [{}], is multibeneficiary response code: [{}]",
```

Comment:
Let's log the transaction id too :)

Reply:
Here we don't have yet the transaction id because it's retrieved from session service response that is called after all payment notices are processed

Reply:
Don't mind the above response, I'll fix it

Reply:
fixed